### DISCUSSION TITLE
I've got stuck on generating the audio, please help 🙏

### DISCUSSION THREAD

**Discussion by adhaputraa** [SWivid/F5-TTS#373] [Q&A]
Could you please help me? After running the program, my system encountered an issue and got stuck on generating audio. Here's the message from the terminal:

C:\Program Files\Python310\lib\site-packages\transformers\models\whisper\generation_[whisper.py:509](javascript:void(0);): FutureWarning: The input name `inputs` is deprecated. Please make sure to use `input_features` instead.
  warnings.warn(
You have passed task=transcribe, but also have set `forced_decoder_ids` to [[1, None], [2, 50360]] which creates a conflict. `forced_decoder_ids` will be ignored in favor of task=transcribe.

And after I close the program, it show another message :
Passing a tuple of `past_key_values` is deprecated and will be removed in Transformers v4.43.0. You should pass an instance of `EncoderDecoderCache` instead, e.g. `past_key_values=EncoderDecoderCache.from_legacy_cache(past_key_values)`.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

What should I do? Thank you for your help.

**Comment by ACoderWorks** [comment]
got the same issue... someone able to help?

**Reply by ACoderWorks** [reply]
SOLVED! Check: https://github.com/SWivid/F5-TTS/issues/356#issuecomment-2454348480

**Comment by sngraam** [comment]
bro you lap cooked help google colab to run
